test/parallel/test-child-process-fork-options.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+
+// This test ensures that fork should parse options
+// correctly if args is undefined or null
+
+const assert = require('assert');
+const { fork } = require('child_process');
+
+const expectedEnv = { foo: 'bar' };
+const cp = fork(fixtures.path('child-process-echo-options.js'), undefined,
+                { env: Object.assign({}, expectedEnv) });
```

Comment:
The test should extend `process.env` not replace it. See https://github.com/nodejs/node/pull/16405.

Reply:
Done.